Apps/Immunization/src/Delegates/RestVaccineStatusDelegate.cs

```diff
@@ -76,7 +76,7 @@ public async Task<RequestResult<PhsaResult<VaccineStatusResult>>> GetVaccineStat
             }
             catch (Exception e) when (e is ApiException or HttpRequestException)
             {
-                this.logger.LogError("Unexpected exception in GetVaccineStatus {Exception}", e);
+                this.logger.LogError("Unexpected exception in GetVaccineStatusAsync {Exception}", e);
```

Comment:
This log statement shouldn't be changed; it's referencing the name of the current method.